src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -131,4 +142,25 @@ private function createAttributesString(array $attributes): string
 
         return $attributeString;
     }
+
+    private function addWebLinkPreloads(Request $request, array $cssLinks): void
+    {
+        $cssPreloadLinks = array_map(fn ($url) => new Link('preload', $url), $cssLinks);
```

Comment:
I'm not sure, but shouldn't the nopush attribute be better set here? So that the css files are not sent again and again, even if they are already in client cache.

From https://developer.chrome.com/blog/early-hints/:
> with HTTP2/Push it was extremely hard to avoid pushing sub-resources that the browser already had. This "over-pushing" effect resulted in a less efficient usage of the network bandwidth, which significantly hindered the performance benefits.

In current Chrome versions it wouldn't make any difference, because Server Push is already disabled there. But afaik other browsers still support Server Push in current versions.

```diff
- $cssPreloadLinks = array_map(fn ($url) => new Link('preload', $url), $cssLinks);
+ $cssPreloadLinks = array_map(fn ($url) => (new Link('preload', $url))->withAttribute('nopush', true), $cssLinks);
```